InventoryInStorePickupShippingApi/Api/IsInStorePickupDeliveryAvailableForCartInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupShippingApi\Api;
```

Comment:
Maybe put it into  `Magento\InventoryInStorePickupShippingApi\Model` ?
As we already have `Magento\InventoryInStorePickupShippingApi\Model\IsInStorePickupDeliveryCartInterface`